### DISCUSSION TITLE
Meeting Notes - 01/04/2023

### DISCUSSION THREAD

**Discussion by orhun** [ratatui/ratatui#123] [Meeting Notes]
# Meeting Notes 01/04/2023

Participation was not as high as expected so we allocated a part of the meeting for user help and questions!
We will discuss more about the upcoming features in the next meeting.

- Feature: Allow word-break in wrapped paragraphs
   - https://github.com/tui-rs-revival/ratatui/pull/33
   - It can be prioritized.

- Feature: New constraint types (expand/shrink based on the rect/area)
  - An issue will be created by @pythops soon.

- Extra/contrib widgets demo by @sophacles
  - https://github.com/tui-rs-revival/ratatui/issues/118
  - Repository: https://github.com/sophacles/extra-widgets
  - We should figure out the next steps.

- Template repository
  - https://github.com/tui-rs-revival/rust-tui-template
  - We might want to add more templates/examples.

**Let me know if I forgot anything!**

**Comment by sayanarijit** [comment]
Sorry couldn't join due to network issues. I've been busy with relocation, so couldn't keep up with the updates. The meeting notes will be helpful. Thank you.

**Comment by dzfrias** [comment]
Was #116 discussed?

**Reply by orhun** [reply]
No, feel free to submit it for the next meeting. We couldn't have that much of feature discussion in this one.
